src/main/kotlin/com/terraformation/backend/seedbank/db/AccessionStore.kt

```diff
@@ -327,7 +326,7 @@ class AccessionStore(
     val organizationId =
         parentStore.getOrganizationId(facilityId) ?: throw FacilityNotFoundException(facilityId)
 
-    if (updated.state?.isV2Compatible != true) {
```

Comment:
Ugh, I hate the `== true` or `!= true` construction that nullability sometimes forces us into.

Reply:
Yeah, I'm not nuts about that either, but I like it _slightly_ better than `?: false`.